src/System.ComponentModel.Primitives/src/System/ComponentModel/DesignOnlyAttribute.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
+    ///    <para>Specifies whether a property can only be set at
+    ///       design time.</para>
+    /// </devdoc>
+    [AttributeUsage(AttributeTargets.All)]
+    public sealed class DesignOnlyAttribute : Attribute, IIsDefaultAttribute
+    {
+        private readonly bool _isDesignOnly = false;
```

Comment:
Thanks - I noticed that while porting the source, but there were so many things that seemed like we could clean it up, we weren't sure where to draw the line....